components/utils/color-utils.css

```diff
@@ -5,7 +5,7 @@
   color: var(--white);
 }
 
-.secondary{
+.secondary {
   background-color: var(--secondary);
   color: var(--white);
```

Comment:
It's not recommended to use color name as variable name. 
Else LGTM!!

Reply:
I have made the changes. Thanks